transmogrifier/sources/datacite.py

```diff
@@ -126,6 +201,27 @@ def create_from_datacite_xml(
                 kind=identifier_xml["identifierType"],
             ),
         ]
+        # language
+        language = xml.metadata.find("language")
+        if language:
+            kwargs["languages"] = [language.string]
+
+        # locations
+        locations = xml.metadata.find_all("geoLocationPlace")
+        for location in locations:
+            kwargs.setdefault("locations", []).append(Location(value=location.string))
+
+        # notes
+        descriptions = xml.metadata.find_all("description")
+        for description in [
+            d
+            for d in descriptions
```

Comment:
`descriptionType` attribute is mandatory in Datacite if the `Description` field is used, so we should do here what we do for other mandatory fields and log a warning if it's not present.